docs/guides/drivers/remote.md

```diff
@@ -14,157 +14,173 @@ $ docker buildx create \
-to the buildkit container without specifying a port:
+   ```console
+   $ ls -lh /home/user/buildkitd.sock
+   srw-rw---- 1 root user 0 May  5 11:04 /home/user/buildkitd.sock
+   ```
 
-```console
-$ docker buildx create \
-  --name remote-container \
-  --driver remote \
-  docker-container://remote-container
-```
+3. Connect Buildx to it using the remote driver:
 
-## Remote Buildkit in Kubernetes
+   ```console
+   $ docker buildx create \
+     --name remote-unix \
+     --driver remote \
+     unix://$HOME/buildkitd.sock
+   ```
 
-In this scenario, we'll create a similar setup to the `kubernetes` driver by
-manually creating a buildkit `Deployment`. While the `kubernetes` driver will
-do this under-the-hood, it might sometimes be desirable to scale buildkit
-manually. Additionally, when executing builds from inside Kubernetes pods,
-the buildx builder will need to be recreated from within each pod or copied
-between them.
+4. List available builders with `docker buildx ls`. You should then see
+   `remote-unix` among them:
 
-Firstly, we can create a kubernetes deployment of buildkitd, as per the
-instructions [here](https://github.com/moby/buildkit/tree/master/examples/kubernetes).
-Following the guide, we setup certificates for the buildkit daemon and client
-(as above using [create-certs.sh](https://github.com/moby/buildkit/blob/v0.10.3/examples/kubernetes/create-certs.sh))
-and create a `Deployment` of buildkit pods with a service that connects to
-them.
+   ```console
+   $ docker buildx ls
+   NAME/NODE           DRIVER/ENDPOINT                        STATUS  PLATFORMS
+   remote-unix         remote
+     remote-unix0      unix:///home/.../buildkitd.sock        running linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/386
+   default *           docker
+     default           default                                running linux/amd64, linux/386
+   ```
 
-Assuming that the service is called `buildkitd`, we can create a remote builder
-in buildx, ensuring that the listed certificate files are present:
+You can switch to this new builder as the default using
+`docker buildx use remote-unix`, or specify it per build using `--builder`:
 
 ```console
-$ docker buildx create \
-  --name remote-kubernetes \
-  --driver remote \
-  --driver-opt cacert=.certs/ca.pem,cert=.certs/client-cert.pem,key=.certs/client-key.pem \
-  tcp://buildkitd.default.svc:1234
+$ docker buildx build . --builder=remote-unix -t test --load
 ```
 
-Note that the above will only work in-cluster (since the buildkit setup guide
-only creates a ClusterIP service). To configure the builder to be accessible
-remotely, you can use an appropriately configured Ingress, which is outside the
-scope of this guide.
-
-To access the service remotely, we can use the port forwarding mechanism in
-kubectl:
+Remember that you need to use the `--load` flag if you want to load the build
+result into the Docker daemon.
+
+## Guide: Remote BuildKit in Docker container
```

Comment:
Not so sure about calling these "Guides" since these will actually appear in the "Manuals" section on https://docs.docker.com :thinking: We could *potentially* split these out into their own files and have them live in the "Guides" section of the docs, but they feel a little short-form and light on details for that kind of setup.

Maybe "Example" could work better? Not sure.